example/src/main/java/io/github/btkelly/gandalf/example/App.java

```diff
@@ -31,8 +32,11 @@ public void onCreate() {
 
         String mockBootstrapUrl = MockWebServerUtil.startMockWebServer(this);
 
+        Log.d("GandalfExample", "Mock server started at " + mockBootstrapUrl);
```

Comment:
I did leave that there as it's only part of the example app and didn't think that should have a log level type setting or a dependency on a special logging class for different build types.